aten/src/ATen/native/sparse/cuda/cuSPARSELtOps.cpp

```diff
@@ -0,0 +1,269 @@
+#include <ATen/cuda/CUDAContext.h>
+#include <ATen/cuda/CUDADataType.h>
+#include <ATen/cuda/CUDASparse.h>
+#include <ATen/cuda/CUDAConfig.h>
+#include <ATen/core/Tensor.h>
+#include <ATen/Dispatch.h>
+#include <c10/core/ScalarType.h>
+#include <c10/cuda/CUDACachingAllocator.h>
+#include <c10/util/Half.h>
+#include <cusparse.h>
+#include <cstdint>
+
+#if !AT_CUSPARSELT_ENABLED()
+
+namespace at{
+namespace native{
```

Comment:
Space after `namespace` and use nested namespaces
```suggestion
namespace at::native {
```